source/gloperate/include/gloperate/stages/base/RasterizationStage.h

```diff
@@ -0,0 +1,105 @@
+
+#pragma once
+
+
+#include <glm/glm.hpp>
```

Comment:
Replace full-project includes with specialized includes for just this header file.